api/cpp/cbindgen.rs

```diff
@@ -598,10 +598,9 @@ fn gen_corelib(
         "CachedRenderingData".to_owned(),
         "    constexpr CachedRenderingData() : cache_index{}, cache_generation{} {}".to_owned(),
     );
-    config.export.body.insert(
-        "EasingCurve".to_owned(),
-        "    constexpr EasingCurve() : tag(Tag::Linear), cubic_bezier{{0,0,1,1}} {}
-    constexpr explicit EasingCurve(EasingCurve::Tag tag, float a, float b, float c, float d) : tag(tag), cubic_bezier{{a,b,c,d}} {}".into()
+    config.export.body.insert( 
+        "EasingCurve".to_owned(), 
+        "    constexpr explicit EasingCurve(EasingCurve::Tag tag = Tag::Linear , float a = 0, float b = 0, float c = 1, float d = 1) : tag(tag), cubic_bezier{{a,b,c,d}} {}".into() 
```

Comment:
```suggestion
        "    constexpr EasingCurve(EasingCurve::Tag tag = Tag::Linear , float a = 0, float b = 0, float c = 1, float d = 1) : tag(tag), cubic_bezier{{a,b,c,d}} {}".into()
```

It shouldn't have been explicit, sorry to have mislead you.

Reply:
No worries...How can I test this locally?  I'm not doing anything with CPP and Slint.

Reply:
You can run the driver test like so.

```
cargo test -p  test-driver-cpp
```

But I'm not sure this test any of the new curves since they are not in the tests but in the gallery.
To build the gallery, you need cmake:

https://github.com/slint-ui/slint/blob/master/docs/building.md#c-api-build